sky/clouds/service_catalog/__init__.py

```diff
@@ -190,10 +190,26 @@ def get_region_zones_for_accelerators(
 
 def check_host_accelerator_compatibility(instance_type: str,
                                          accelerators: Optional[Dict[str, int]],
-                                         zone: Optional[str] = None,
                                          clouds: CloudFilter = None) -> None:
-    """GCP only: Check if host VM type is compatible with the accelerators."""
+    """GCP only: Check if host VM type is compatible with the accelerators.
```

Comment:
Can we add https://github.com/skypilot-org/skypilot/pull/989#issuecomment-1232479501 to this func and the next func (L207+)? It's great explanation on why these two funcs are structured this way.

Reply:
Added.